components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -165,10 +165,14 @@ export class Panel extends React.Component<Props, State> {
     }
   }
 
-  openRewardsPage () {
+  openRewardsPage (id?: string) {
```

Comment:
I wasn't sure what the behavior of the "Turn on Ads" button should be exactly. Right now it just navigates them to the Rewards page.

Do we want it to turn on the Ads service right from the panel? I could enable ads via `chrome.braveRewards.saveAdsSetting`, then redirect the user to `brave://rewards` for further interaction.

cc: @mandar-brave